Projects/GameOfLife/tests/test_gameOfLife.py

```diff
@@ -0,0 +1,25 @@
+## File to be run to test - currently two test pass and one fails
+## ran using pytest test_gameOfLife.py on console
+
+import sys
+import os
+
+from GameOfLife import GameOfLife
+
+test_obj = GameOfLife(3, 1, 0)
+
+
+def test_initial_grid_creation():
+    test_obj.make_grid()
+    assert test_obj.grid == [[1, 0, 1], [1, 0, 1], [1, 0, 1]]
```

Comment:
assertion should be done in this way.
calling the method .make_grid() does not return any value. asserting the result of that method will compare None with your expected result
what you're trying to do here is asserting that the object modified by the method is as expected